software_installation.md

```diff
@@ -53,3 +58,37 @@ We will create a MiniConda environment with the following Python packages:
    It should look like this: ![](./images/choose_kernel.png)
 
 5. Test the environment and make sure that it works as expected.
+
+
+### Environment creation: R
+ 
+For R, let's create an environment with following packages, and let's name it r41env:
+
+- `R 4.1`
+- `dplyr`
+- `data.table`
+- `ggplot2`
+- `tidyr`
+
+> **_NOTE:_** Instead of creating a new environment you can also introduce
+  an existing environment to your JupyterHub. In this case you can skip the
+  creation of the environment.
+
+1. Create the new R environment: 
+  ```
+  conda create -n r41env r-base=4.1 r-dplyr r-data.table r-ggplot2 r-tidyr -c conda-forge
```

Comment:
Also the "jupyter_client" package needs to be added here (without this the IRkernel installation step will fail).